app/views/spree/order_mailer/cancel_email.html.haml

```diff
@@ -1,11 +1,33 @@
-%h3
-  = t(".customer_greeting", name: @order.bill_address.firstname)
-%h4
-  = t(".instructions")
-%span.clear
+%table.social.white-bg{:width => "100%"}
+  %tr
+    %td
+      %table.column{:align => "left"}
+        %tr
+          %td
+            %h3
+              = t(".customer_greeting", name: @order.bill_address.firstname)
+            %h4
+              = t(".instructions_html", distributor: @order.distributor.name )
+            %img.float-left{:src => "#{@order.distributor.logo.url(:medium)}"}/
```

Comment:
this is copy pasted from the confirmation email, right?
in this case you dont use a second table for the image, you have it immediately below the instructions. why?
I think that if you dont need a second table (depends on previous question), you will not need a table inside a table like you have now.

Reply:
I agree with you. I didn't pay attention to this. The image is right below the instructions, so I will fix this.